src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.spec.ts

```diff
@@ -36,4 +36,236 @@ describe('InstructorCoursesPageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with all courses in course stats', () => {
+    const activeCourses: any[] = [
+      {
+        id: 'CS3281',
+        name: 'Modifiable Students and Courses',
+        createdAt: '2018-11-05T08:15:30',
+        canModifyCourse: true,
+        canModifyStudent: true,
+      },
+      {
+        id: 'CS3282',
+        name: 'Nothing modifiable',
+        createdAt: '2019-02-02T08:15:30',
+        canModifyCourse: false,
+        canModifyStudent: false,
+      },
+      {
+        id: 'CS2103',
```

Comment:
I think this is unnecessary because from the page, `canModifyStudent` and `canModifyCourse` render unrelated things, so the 2 test cases above should be enough. What do you think?

Reply:
Hmm I suppose that makes sense. But no harm in having them once added, so I'll leave it as it is?

Reply:
Hmm then it makes the snapshot unnecessary longer than needed, making it harder to read... I think it'll be easier to spot mistakes in the rendering when the snapshot is shorter, so perhaps it's better if the unnecessary test cases are removed

Reply:
Hmm aite I'll remove them